docs/src/man/basics.md

```diff
@@ -0,0 +1,1505 @@
+# First Steps with DataFrames
+
+The first and important part to work with DataFrames.jl, is to install the `DataFrames`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
+
+```julia
+julia> ] # `]` should be pressed 
+
+(@v1.6) pkg>  add DataFrames
+```
+
+To make sure everything works as expected, try to load the package and if you have the time execute
+its test suits. It can take around 2726.057766 seconds (770.99 k allocations: 59.873 MiB, 0.07% gc time).
```

Comment:
```suggestion
If you want to make sure everything works as expected you could run the tests bundled
with DataFrames.jl, but be warned that it will take more than 30 minutes.
```